xml/obs_source_service.xml

```diff
@@ -410,7 +410,7 @@
 
  <sect1 xml:id="sec.obs.sserv.remove">
   <title>Removing a Source Service</title>
-  <para> Sometimes it is useful to continued to work on generated files
+  <para> Sometimes it is useful to continue work on generated files
```

Comment:
Nitpick: maybe "continue working" fits better?

Thanks for reviewing the docs.

Reply:
I think it should be `to continue working` or `to continue to work`. I think `to continue work` is wrong, except if `work` is a noun, but here it is a verb.

Reply:
I thought it was using as a noun, since there was a identical string on `xml/obs_ag_source_service.xml`. I went ahead and changed both strings.